### PR TITLE
XS✔ ◾ Improve-stale-pr-check

### PR DESCRIPTION
**Pull request by wicksipedia**
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

email `RE: [GitHub] You've used 75% of included services for the SSWConsulting account`

> 2. What was changed?

changed stale workflow from code to an action from GitHub
changed schedule to weekly
added a dependabot config

sample run (includes more logging): https://github.com/SSWConsulting/SSW.GitHub.Template/actions/runs/8385929456

> 3. Did you do pair or ~mob~ ensemble programming?

no

### CODE REVIEW

**Review by danielmackay — APPROVED** [review]
LGTM

File: .github/workflows/pr-manage-stale.yml [hunk 1]
```diff
@@ -4,157 +4,38 @@
...
-    branches: [main]
-
-  # schedule:
-  #   # run every 2nd hour
-  #   # https://crontab.guru/#0_0/2_*_*_*
-  #   - cron: "0 0/2 * * *"
-
+  schedule:
+    # https://crontab.guru/#0_20_*_*_0 - At 20:00 on Sunday (UTC)
+    - cron: '0 20 * * 0'
```
Comment (GordonBeeming) [inline .github/workflows/pr-manage-stale.yml:9]: Should we not run this at least once a day?  
Instead of once a week?

**Comment by GordonBeeming** [comment]
@wicksipedia 
1. is the intent to change PR Stale just for private repos or public too?
2. If the standard is different, we should update https://www.ssw.com.au/rules/standard-set-of-pull-request-workflows/

**Comment by wicksipedia** [comment]
> 1. is the intent to change PR Stale just for private repos or public too?

we should ideally do the same for both
the original flow worked well for a particular client since we needed to gamify smaller prs to change behaviour. Not really a requirement for _all_ projects

If we wanted to return to that setup, it would be better to have an org level action scan all repos and age prs as required - it would be efficient with action minute usage.

> 2. If the standard is different, we should update https://www.ssw.com.au/rules/standard-set-of-pull-request-workflows/

Only the label screenshots need to be updated, not sure if it's a must have right now. I'll update them once we have a pr that is stale by the new standard - https://github.com/SSWConsulting/SSW.GitHub.Template/pull/24

**Pull request merged**